runtime/builtin_types_test.go

```diff
@@ -422,3 +423,63 @@ func TestBuiltinSetAttr(t *testing.T) {
 		}
 	}
 }
+
+func TestRawInput(t *testing.T) {
+	fun := newBuiltinFunction("TestRawInput", func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+		// Create a fake Stdin for input test.
+		stdinFile, _ := ioutil.TempFile(os.TempDir(), "prefix")
+		stdinFile.Write([]byte("hello grumpy\n"))
+		stdinFile.Close()
```

Comment:
I don't think you need to reopen stdinFile. You should be able to read what you've written. You may need to Sync() (and possibly Seek() to the beginning, but I don't think so).